app/src/main/res/drawable/dialog_insert_character_id_background.xml

```diff
@@ -0,0 +1,19 @@
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <item>
+        <shape android:shape="rectangle">
+            <size
+                android:width="300dp"
```

Comment:
did you try this in smaller devices?

Reply:
Mau, I make some changes and now it works good in a smaller device! Fixed!